src/formio/components/currency.ts

```diff
@@ -0,0 +1,29 @@
+import {InputComponentSchema, MultipleCapable, PrefillConfig} from '..';
+
+type Validator = 'required' | 'min' | 'max';
+type TranslatableKeys = 'label' | 'description' | 'tooltip' | 'suffix';
+
+export type CurrencyInputSchema = InputComponentSchema<number, Validator, TranslatableKeys>;
+
+/**
+ * @group Form.io components
+ * @category Base types
+ */
+export interface BaseCurrencyComponentSchema
+  extends Omit<CurrencyInputSchema, 'hideLabel'>,
+    PrefillConfig {
```

Comment:
prefill is not used with currency

```suggestion
    {
```